src/legacy/button.js

```diff
@@ -64,7 +64,7 @@ function renderButton(id, { container, locale, type, color, shape, size }) : Zal
 }
 
 // eslint-disable-next-line flowtype/no-mutable-array
-export function renderButtons(id : string, buttons : Array<Object>) : ZalgoPromise<Array<Object>> {
+export function renderButtons(id : string, buttons : Array<Object>) : ZalgoPromise<$ReadOnlyArray<Object>> {
```

Comment:
This resolves the following flow error:

```
Error --------------------------------------------------------------------------------------- src/legacy/button.js:69:12

Cannot return `ZalgoPromise.map(...)` because read-only array type [1] is incompatible with array type [2] in type
argument `R` [3].

   src/legacy/button.js:69:12
                   v------------------------------------
    69|     return ZalgoPromise.map(buttons, button => {
```

Ex: https://github.com/paypal/paypal-checkout-components/runs/1559689002?check_suite_focus=true#step:6:101